setup.py

```diff
@@ -31,7 +31,7 @@ def find_stub_files(name: str) -> List[str]:
 ]
 
 extras_require = {
-    "compatible-mypy": ["mypy>=0.991,<1.0"],
+    "compatible-mypy": ["mypy>=1.0,<1.1"],
```

Comment:
Do we need to bump the minimum, or is that the policy? Seems like there's no braking changes here?

Reply:
This only affects those that use `django-stubs[compatible-mypy]`, yes we bump lower bound in that case.